### PR TITLE
Added documentation explaining contrib and outlining process around it

### PR DESCRIPTION
**Pull request by reinecke**
Adds a README.md to the contrib area to explain the break-out process in-progress.

### CODE REVIEW

**Comment by jminor** [comment]
This is great! Could you put a link to it in place of the "Registering Your Contrib Adapter" here: https://github.com/AcademySoftwareFoundation/OpenTimelineIO/blob/main/docs/tutorials/write-an-adapter.md

**Comment by reinecke** [comment]
@jminor good note on the other adapter page - I opted to reproduce the instructions a bit there so when the contrib area migration is done we won't have to update any links over there. I also reshuffled/touched it up so that it highlighted the preferred route of cloning the template repo.

File: contrib/README.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# OpenTimelineIO Contrib Area
+
+The contrib area was a place to host adapters and application plugins submit
+from the community, that have challenging external dependencies, or may not
+have the same level of support as the core.
```
Comment (meshula) [inline contrib/README.md:5]:
```suggestion
The contrib area was a place to host adapters and application plugins submitted
by the OpenTimelineIO community. Those contributions may have involved challenging external 
dependencies, and may not have the same level of support as the core.
```

File: contrib/README.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+The team is in the process of phasing out this area in favor of using individual
+repos within the
+[OpenTimelineIO GitHub Organization](https://github.com/OpenTimelineIO/).
+
+New contrib submissions should be created in standalone repos and can be later
+migrated to the OTIO GitHub organization after review.
```
Comment (meshula) [inline contrib/README.md:12]:
```suggestion
New contributions should be created in standalone repos, and those can later be
migrated to the OTIO GitHub organization after review.
```

**Review by meshula — COMMENTED** [review]
minor wordsmithing

**Review by meshula — APPROVED** [review]

**Pull request merged**